js/src/sdk/actionRegistry.ts

```diff
@@ -135,6 +135,15 @@ export class ActionRegistry {
     );
   }
 
+  async getToolName({
+    action,
+  }: {
+    action: string;
```

Comment:
if there is no action sent, throw an error

Reply:
Added the condition inside [`createAction()`](https://github.com/ComposioHQ/composio/pull/1539/commits/6370e77aa9e403f6b46ae097f56cc3a2ebb2fd0d). LMK if this works

Reply:
Done